api/spec/smithy/model/types/Cluster.smithy

```diff
@@ -30,6 +30,8 @@ structure ClusterInfoSummary {
     @required
     @documentation("Status of the cluster infrastructure.")
     clusterStatus: ClusterStatus,
+    @documentation("Scheduler of the cluster.")
+    scheduler: Scheduler,
```

Comment:
do we need `@required` here?

Reply:
Didn't add `@required` because ClusterInfoSummary is also used for CreateCluster, DeleteCluster, UpdateCluster. Now added `@requried` as we add Scheduler also to  CreateCluster, DeleteCluster, UpdateCluster result